src/node.cc

```diff
@@ -2177,7 +2177,15 @@ void FatalException(Isolate* isolate,
 
   if (false == caught->BooleanValue()) {
     ReportException(env, error, message);
-    exit(1);
+    if (abort_on_uncaught_exception) {
+#ifdef _WIN32
+      raise(SIGABRT);
+#else
+      abort();
+#endif
```

Comment:
Would it not be safer/clearer to to use a inline function named `node_abort` or `node_fatal` instead of an unsafe macro with a not really unique name?